### ISSUE TITLE
Variables.scss Multiplication error

### ISSUE THREAD

**Issue by huwrowlands** [huwrowlands/imperavi-kube-5-scss#1]
Getting this error upon saving the file:

```
Error: invalid operands for multiplication.
on line 31 of Users/huw/Projects/WordPress/MyProjects/holidaycaravans/wp-content/themes/hciw/assets/scss/kube/variables.scss
>> $grid-gap: (unit($grid-gutter)/($grid-width/100))*1%;
```

**Comment by thatgibbyguy** [comment]
are you using `libsass` to compile this or something else?

**Comment by huwrowlands** [comment]
Yes libasass, sorry.

I'm guessing the main issue is the way less and sass do things differently.

**Comment by thatgibbyguy** [comment]
Yeah, I ran into this with ply too. You can see what I had to do to fix it here - https://github.com/thatgibbyguy/ply/blob/master/scss/components/_variables.scss#L9

Basically with libsass you have specify what type of number you're working with and also make sure that the number you're working with will pass smoothly when you invoke your mixin. I'm pretty sure rubysass doesn't mind what you're doing, but libsass definitely does.

**Comment by thatgibbyguy** [comment]
Actually a better example would be line 26 of that link. Notice I do `$base-font-size + 0em;` - that sets it to ems for everything else.

**Comment by huwrowlands** [comment]
Thanks for the replies. I still couldn't manage to figure out how to rewrite it sadly.

**Comment by thatgibbyguy** [comment]
No problem, sorry for not being able to do more with this. Been swamped.
